File: 2-Anno/Linguaggi/Java/Pratica/Simulazioni/N40/Furgone.java
package N40;

public class Furgone extends Veicolo{
	
	private String categoria;
	private int numeroPosti;
	private boolean vanoCarico;

	public Furgone(int codice, String tipo, String targa, String modello, String marca, double costoGiornaliero, String categoria, int numeroPosti, boolean vanoCarico) {
		super(codice, tipo, targa, modello, marca, costoGiornaliero);
		this.categoria = categoria;
		this.numeroPosti = numeroPosti;
		this.vanoCarico = vanoCarico;
	}

	public String toString() {
		return super.toString() + "\t-\t-\t" + categoria + "\t" + numeroPosti + "\t" + vanoCarico;
	}
}

File: 2-Anno/Linguaggi/Python/Pratica/Simulazioni/N40/Furgone.py
from Veicolo import Veicolo

class Furgone(Veicolo):
    
	def __init__(self, codice, tipo, targa, modello, marca, costoGiornaliero, categoria, nPosti, vanoCarico) -> None:
		super().__init__(codice, tipo, targa, modello, marca, costoGiornaliero)
		self._categoria = categoria
		self._nPosti = nPosti
		self._vanoCarico = vanoCarico

	def __str__(self) -> str:
		return super().__str__() + "\t-\t-\t" + str(self._categoria) + "\t" + str(self._nPosti) + "\t" + str(self._vanoCarico)
